src/GraphQL/Types/ArgumentInformation.cs

```diff
@@ -121,7 +121,10 @@ public ArgumentInformation(ParameterInfo parameterInfo, Type? sourceType, FieldT
         /// The delegate must be of the type
         /// <see cref="Func{T, TResult}">Func</see>&lt;<see cref="IResolveFieldContext"/>, <typeparamref name="TParameterType"/>&gt;
         /// where <typeparamref name="TParameterType"/> matches <see cref="ParameterInfo">ParameterInfo</see>.<see cref="ParameterInfo.ParameterType">ParameterType</see>.
+        /// <br/><br/>
+        /// As of .NET 7, this method cannot be used in AOT compilation scenarios.
```

Comment:
Why?

Reply:
It threw some obscure error; I did not investigate.  It seemed to me that the necessary reflection (e.g. the `Type` class) was not preserved with/for the associated lambda function that was passed in.  Probably a .NET 7 bug, but again, I did not investigate much.

Reply:
OK. Then just add your comment as is in the method body with TODO marker.